vizro-core/src/vizro/actions/_actions_utils.py

```diff
@@ -177,8 +177,10 @@ def _create_target_arg_mapping(dot_separated_strings: List[str]) -> Dict[str, Li
 def _update_nested_graph_properties(graph_config: Dict[str, Any], dot_separated_string: str, value: Any):
```

Comment:
```suggestion
def _update_nested_graph_properties(graph_config: Dict[str, Any], dot_separated_string: str, value: Any) -> Dict[str, Any]:
```
This should probably have been there before